scripts/playbook.yml

```diff
@@ -29,11 +31,15 @@
         - cmake
         - python-psycopg2
         - libselinux-python
-        - git
         - bash-completion
         - vim
-    - name: install bower package manager
-      npm: name=bower global=yes
+    - name: install npm packages needed
+      npm: name={{ item }} global=yes
+      with_items:
+        - bower
+        - yarn
+        - gulp-cli
+        - webpack
```

Comment:
Right now, ops UI really needs only `bower` and `npm`.

SUI however needs `webpack` and `yarn` - those are only needed to build SUI, not to use it, but since this is a **dev** setup, I think they are needed here.

In the short future, ops UI will also need `webpack` and `yarn`.

`gulp-cli` is probably not needed at all, since SUI stopped using gulp and ops UI never used it.